### PR TITLE
Added Location, Vector, world and some basic members

### PR DESCRIPTION
**Pull request by Digot**

### CODE REVIEW

File: gomint-server/src/main/java/io/gomint/server/entity/Entity.java [hunk 1]
```diff
@@ -15,7 +17,13 @@
  */
 @Getter
 public class Entity {
-    private double x;
-    private double y;
-    private double z;
+
+    @Getter
```
Comment (geNAZt) [inline gomint-server/src/main/java/io/gomint/server/entity/Entity.java]: Getters should be inlined if written with Lombok

File: gomint-server/src/main/java/io/gomint/server/entity/Entity.java [hunk 2]
```diff
@@ -15,7 +17,13 @@
...
+    private Location location;
+
```
Comment (geNAZt) [inline gomint-server/src/main/java/io/gomint/server/entity/Entity.java]: Also no newline between variable declerations

File: gomint-server/src/main/java/io/gomint/server/world/Location.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+import lombok.Getter;
+import lombok.Setter;
+
+import java.text.MessageFormat;
+
+/**
+ * Created by Markus on 21.10.2015.
+ */
+public class Location implements Cloneable<Location> {
+
```
Comment (geNAZt) [inline gomint-server/src/main/java/io/gomint/server/world/Location.java]: No newline at the top/bottom of classes

File: gomint-server/src/main/java/io/gomint/server/world/Location.java [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    @Getter @Setter
+    private float x;
```
Comment (geNAZt) [inline gomint-server/src/main/java/io/gomint/server/world/Location.java]: Location should be a double aswell to support Vector based subs like .toVector() or new Location( Vector vector, String worldName );

File: gomint-server/src/main/java/io/gomint/server/world/Vector.java [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+    private double x;
+
+    @Getter @Setter
+    private double y;
+
+    @Getter @Setter
+    private double z;
+
+    public Vector(){
+        x = 0;
```
Comment (geNAZt) [inline gomint-server/src/main/java/io/gomint/server/world/Vector.java]: Instance variables should be used with this. in as much places as possible

File: gomint-server/src/main/java/io/gomint/server/world/Vector.java [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+    }
+
+    public Vector(float x, float y, float z){
+        this.x = x;
+        this.y = y;
+        this.z = z;
+    }
+
+    public Vector add(Vector other){
+        x += other.getX();
```
Comment (geNAZt) [inline gomint-server/src/main/java/io/gomint/server/world/Vector.java]: Vectors should be immutable

**Comment by Digot** [comment]
Codestyle has been fixed

**Pull request merged**